content/en/docs/concepts/configuration/manage-resources-containers.md

```diff
@@ -161,56 +179,54 @@ When you create a Pod, the Kubernetes scheduler selects a node for the Pod to
-  or 2 is used as the value of the
-  [`--cpu-shares`](https://docs.docker.com/engine/reference/run/#cpu-share-constraint)
-  flag in the `docker run` command.
-
-- The `spec.containers[].resources.limits.cpu` is converted to its millicore value and
-  multiplied by 100. The resulting value is the total amount of CPU time in microseconds
-  that a container can use every 100ms. A container cannot use more than its share of
-  CPU time during this interval.
-
-  {{< note >}}
-  The default quota period is 100ms. The minimum resolution of CPU quota is 1ms.
-  {{</ note >}}
-
-- The `spec.containers[].resources.limits.memory` is converted to an integer, and
-  used as the value of the
-  [`--memory`](https://docs.docker.com/engine/reference/run/#/user-memory-constraints)
-  flag in the `docker run` command.
-
-If a Container exceeds its memory limit, it might be terminated. If it is
-restartable, the kubelet will restart it, as with any other type of runtime
-failure.
-
-If a Container exceeds its memory request, it is likely that its Pod will
-be evicted whenever the node runs out of memory.
-
-A Container might or might not be allowed to exceed its CPU limit for extended
-periods of time. However, it will not be killed for excessive CPU usage.
-
-To determine whether a Container cannot be scheduled or is being killed due to
-resource limits, see the
-[Troubleshooting](#troubleshooting) section.
+## How Kubernetes applies resource requests and limits {#how-pods-with-resource-limits-are-run}
+
+When the kubelet starts a container as part of a Pod, the kubelet passes that container's
+requests and limits for memory and CPU to the container runtime.
+
+On Linux, the container runtime typically configures
+kernel {{< glossary_tooltip text="cgroups" term_id="cgroup" >}} that apply and enforce the
+limits you defined.
+
+- The CPU limit defines a hard ceiling on how much CPU time that the container can use.
+  During each scheduling interval (time slice), the Linux kernel checks to see if this
+  limit is exceeded; if so, the kernel waits before allowing that cgroup to resume execution.
+- The CPU request typically defines a weighting. If several different containers (cgroups)
+  want to run on a contended system, workloads with larger CPU requests are allocated more
+  CPU time than workloads with small requests.
+- The memory request is mainly used during (Kubernetes) Pod scheduling. On node that uses
+  cgroups v2, the container runtime might use the memory request as a hint to set
+  `memory.min` and `memory.low`.
+- The memory limit defines a memory limit for that cgroup. If the container tries to
+  allocate more memory than this limit, the Linux kernel out-of-memory subsystem activates
+  and, typically, intervenes by stopping one of the processes in the container that tried
+  to allocate memory. If that process is the container's PID 1, and the container is marked
```

Comment:
What's PID 1? The container's first process?

Reply:
PID 1 is the primary process for a POSIX system. There is a strong convention that if PID 1 (from the container's point of view) fails or shuts down, then the container runtime should terminate the container.

Maybe that could be a glossary entry? (separate PR)

Reply:
+1 for glossary entry :) I had to google it